### PR TITLE
create db, create migrater command, create models

### PR DESCRIPTION
**Pull request by nyrbrg**

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"database/sql"
+	"fmt"
+)
+
+func Connect(connection string) (*sql.DB, error) {
+	db, err := sql.Open("postgres", connection)
+
+	if err != nil {
+		panic(err)
```
Comment (PumpkinSeed) [inline database/database.go]: `return nil, err`

Never use panic in package, it can cause unpredictable behaviour.

File: database/database.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	}
+
+	err = db.Ping()
+	if err != nil {
+		panic(err)
```
Comment (PumpkinSeed) [inline database/database.go]: `return nil, err`

File: database/database.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+func Connect(connection string) (*sql.DB, error) {
+	db, err := sql.Open("postgres", connection)
+
+	if err != nil {
+		panic(err)
+	}
+
+	err = db.Ping()
```
Comment (PumpkinSeed) [inline database/database.go]: db.Ping isn't necessary

File: database/database.go [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Successfully connected!")
```
Comment (PumpkinSeed) [inline database/database.go]: Use log package for logging.

log.Println(...)

**Review by PumpkinSeed — CHANGES_REQUESTED** [review]
Nice job. Solve the requested changes and I will write the next task.

File: database/database.go (hunk 1 above)
Comment (nyrbrg) [reply database/database.go]: I took it out.

File: database/database.go (hunk 2 above)
Comment (nyrbrg) [reply database/database.go]: I took it out.

File: database/database.go (hunk 3 above)
Comment (nyrbrg) [reply database/database.go]: I took it out.

File: database/database.go (hunk 4 above)
Comment (nyrbrg) [reply database/database.go]: Update logging.

**Review by PumpkinSeed — APPROVED** [review]

**Pull request merged**